plugins/modules/rds_engine_versions_info.py

```diff
@@ -0,0 +1,377 @@
+                "supported_engine_modes": [
+                    "provisioned"
+                ],
+                "supports_babelfish": true,
+                "supports_global_databases": true,
+                "supports_integrations": false,
+                "supports_local_write_forwarding": true,
+                "supports_parallel_query": false
+            }
+        ]
+    supported_timezones:
+        description:
+        - A list of the time zones supported by this engine for the Timezone parameter of the CreateDBInstance action.
+        type: list
+        elements: dict
+        sample: [
+            {"TimezoneName": "xxx"}
+        ]
+    exportable_log_types:
+        description:
+        - The types of logs that the database engine has available for export to CloudWatch Logs.
+        type: list
+        elements: str
+    supports_log_exports_to_cloudwatchLogs:
+        description:
+        - Indicates whether the engine version supports exporting the log types specified by ExportableLogTypes to CloudWatch Logs.
+        type: bool
+    supports_read_replica:
+        description:
+        - Indicates whether the database engine version supports read replicas.
+        type: bool
+    supported_engine_modes:
+        description:
+        - A list of the supported DB engine modes.
+        type: list
+        elements: str
+    supported_feature_names:
+        description:
+        - A list of features supported by the DB engine.
+        type: list
+        elements: str
+        sample: [
+            "Comprehend",
+            "Lambda",
+            "s3Export",
+            "s3Import",
+            "SageMaker"
+        ]
+    status:
+        description:
+        - The status of the DB engine version, either available or deprecated.
+        type: str
+    supports_parallel_query:
+        description:
+        - Indicates whether you can use Aurora parallel query with a specific DB engine version.
+        type: bool
+    supports_global_databases:
+        description:
+        - Indicates whether you can use Aurora global databases with a specific DB engine version.
+        type: bool
+    major_engine_version:
+        description:
+        - The major engine version of the CEV.
+        type: str
+    database_installation_files_s3_bucket_name:
+        description:
+        - The name of the Amazon S3 bucket that contains your database installation files.
+        type: str
+    database_installation_files_s3_prefix:
+        description:
+        - The Amazon S3 directory that contains the database installation files.
+        type: str
+    db_engine_version_arn:
+        description:
+        - The ARN of the custom engine version.
+        type: str
+    kms_key_id:
+        description:
+        - The Amazon Web Services KMS key identifier for an encrypted CEV.
+        type: str
+    create_time:
+        description:
+        - The creation time of the DB engine version.
+        type: str
+    tag_list:
```

Comment:
@tremble @abikouo What do you think about renaming this field to tag rather than tag_list to be consistent with our collection?

Reply:
Not just that, it should also be returning a normal tag "dict" rather than the boto3 style tag-list

Reply:
updated